smarts/core/remote_agent_buffer.py

```diff
@@ -185,9 +188,22 @@ def acquire_remote_agent(self, retries=3) -> RemoteAgent:
 
 
 def spawn_local_zoo_manager(port):
-    manager = Process(target=zoo_manager.serve, args=(port,))
-    manager.start()
-    return manager
+    cmd = [
+        sys.executable,  # Path to the current Python binary.
+        str(
+            (pathlib.Path(__file__).parent.parent / "zoo" / "manager.py")
+            .absolute()
+            .resolve()
+        ),
+        "--port",
+        str(port),
+    ]
+
+    manager = subprocess.Popen(cmd)
+    if manager.poll() == None:
```

Comment:
Is it ok to do this immediately?  Or might the call to manager.py have some delay before it could potentially fail?

Reply:
Thus far, it looks okay to check whether the subprocess started properly soon after calling `subprocess.Popen(cmd)`. If the manager server inside `manager.py` fails later, we should see some grpc communication errors.